lib/modules/uscore_v3.1.0/us_core_pulse_oximetry_sequence.rb

```diff
@@ -607,19 +607,9 @@ def patient_ids
             * component.code.coding.code
             * component.code.coding.code
             * component.dataAbsentReason
-            * component.valueBoolean
-            * component.valueCodeableConcept
-            * component.valueDateTime
-            * component.valueInteger
-            * component.valuePeriod
             * component.valueQuantity
             * component.valueQuantity
             * component.valueQuantity
```

Comment:
Can we slap a `uniq` on the code that generates this to get rid of all the dupes?

Reply:
Got rid of the duplicates. There shouldn't be any duplicates once we implement must support for subelements of slices, but this makes things less confusing for now.